enterprise/websvc.jaxwsmodel/src/org/netbeans/modules/websvc/api/jaxws/wsdlmodel/java/WsdlJavaMethod.java

```diff
@@ -76,15 +76,15 @@ public JavaParameter getParameter(String paramName) {
 
     public List<JavaParameter> getParametersList() {
         if(this.parameters == null) {
-            this.parameters = new ArrayList<JavaParameter>();
+            this.parameters = new ArrayList<>();
             for(com.sun.tools.ws.processor.model.java.JavaParameter p:this.method.getParametersList()) {
                 this.parameters.add(new WsdlJavaParameter(p));
             }
         }
         return this.parameters;
     }
 
-    public Iterator getExceptions() {
+    public Iterator<String> getExceptions() {
```

Comment:
Would it make sense to update the return value of the interface (module `websvc.jaxwsmodelapi`)?

Reply:
It would make sense to update the interface. The problem is that would require the change to be done to IndexElementImpl.java.

So we're going to have to leave it alone for the time being..